packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkedForest.spec.ts

```diff
@@ -53,79 +48,58 @@ describe("ChunkedForest", () => {
 	});
 
 	it("doesn't copy data when capturing and restoring repair data", () => {
```

Comment:
How are the changes to this test related to the rest of this PR?

Reply:
This test had a lot of needless dependencies (ex: doing most of the end to end editing process), which made it (and another test I fixed) the only users of a bunch of helper logic to replicate the old code thats deleted. These changes reduce its dependencies and simplify whats done here.